### ISSUE TITLE
add improvements to accessibility rule to site

### ISSUE THREAD

**Issue by britt6612** [grommet/hpe-design-system#4783]
Add the following to the site:
updates to improve the a11y rule communication:

- We are keeping the notification below the WCAG compliance title, about the conditional tests as is. 

- In the overview, tests will be shown as follows:
1. Passed (ok icon)
2. Failed (critical icon)
3. Conditional (info icon)
4. AAA Failed (warning icon) 

![Image](https://github.com/user-attachments/assets/0d86eb00-2cfb-4aa9-8f85-d2f74e90322b)

- Failed AAA tests will have the `AAA Failed` status with a warning icon:

![Image](https://github.com/user-attachments/assets/3b10be58-4566-438e-b18d-033409bb8469)

- In the tests accordion, we'll order the tests by a11y principle, and inside each principle, tests will be displayed in the following order:
1.  Critical Failed tests (A/AA)
3. Conditional tests
2. Failed Advisory tests (AAA)
4. Passed tests (A/AA/AAA)

**Comment by britt6612** [comment]
completed by 
https://github.com/grommet/hpe-design-system/pull/4788
